chromeextension/background.js

```diff
@@ -1,9 +1,21 @@
 var backgroundConfig = {
     mojitoBaseUrl: 'http://localhost:8080/',
+    experimentUrl: 'http://localhost:8080/copytune/',
     enabled: false,
     headerName: 'X-Mojito-Ict',
     headerValue: 'on',
-    removeTagsBlock: true
+    removeTagsBlock: true,
+    experimentButtons: {
```

Comment:
let's make this a little bit more generic. `experimentButtons` --> `actionButtons`